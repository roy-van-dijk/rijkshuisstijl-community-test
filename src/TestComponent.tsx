import { Button } from '@rijkshuisstijl-community/components-react';

export function TestComponent() {

  return (
    <Button>Test button!</Button>
  )
} 