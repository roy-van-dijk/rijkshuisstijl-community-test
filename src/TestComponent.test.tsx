import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import userEvent from '@testing-library/user-event';
import { TestComponent } from './TestComponent';

describe('<TestComponent>', () => {
  it('renders', async () => {
    const { container } = render(<TestComponent />);

    await userEvent.click(screen.getByText('Test button!'));

    expect(container).toMatchSnapshot();
  });
});