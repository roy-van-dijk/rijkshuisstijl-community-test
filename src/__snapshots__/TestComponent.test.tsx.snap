// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TestComponent> renders 1`] = `
<div>
  <button
    class="utrecht-button"
    type="button"
  >
    Test button!
  </button>
</div>
`;
